dropshot/src/handler.rs

```diff
@@ -377,6 +379,19 @@ where
     }
 }
 
+impl std::fmt::Display for HandlerError {
+    fn fmt(&self, f: &mut Formatter) -> FmtResult {
+        write!(
+            f,
+            "{}",
+            match self {
+                Self::Handler { ref message, .. } => message,
+                Self::Dropshot(ref e) => &e.external_message,
+            }
+        )
+    }
+}
+
```

Comment:
I think this is for tokio-tracing and could potentially be dropped or separated into a dedicated commit. Really all the tokio-tracing bits should be a separate commit now that I think about it.